oop/homework.py

```diff
@@ -32,38 +32,68 @@ class Cat:
       return text like this: f"Your cat ran {ran_km} kilometers"
 
     * Implement get_saturation_level and return saturation_level
-      if saturation_level eq 0 return text like this: "Your cat is died :("
+      if saturation_level eq 0 return text like this: "Your cat is dead :("
 
     * Implement get_average_speed and return average_speed
 
     """
 
     def __init__(self, age):
-        pass
+        self.age = age
+        self.average_speed = self._set_average_speed()
+        self.saturation_level = 50
 
     def eat(self, product):
-        pass
+        if product == 'fodder':
+            self._increase_saturation_level(10)
+        elif product == 'apple':
+            self._increase_saturation_level(5)
+        elif product == 'milk':
+            self._increase_saturation_level(2)
```

Comment:
you can create separate dict with name: value pairs. and here just check if input in those list and get the value

Reply:
thanks. will fix it